README.md

```diff
@@ -20,10 +20,17 @@ This is a repository for MathShare backend server. MathShare is a step by step e
 First, we need to run s PostgreSQL database, the most convenient way to run it is to use Docker.
 The following command will create a Docker container with all properties configured (database and user).
 
+##### DB initialization
```

Comment:
Do we need any of this - why not just use the compose.yml ?

Reply:
So far, we don't have docker environment configured. This instruction is only for local development and testing.

Reply:
Isn't that a different PR that is already waiting to merge?